OracleSOASuite/dockerfiles/12.2.1.2/Dockerfile.12.2.1.3

```diff
@@ -0,0 +1,47 @@
+#
+# Copyright (c) 2016-2017 Oracle and/or its affiliates. All rights reserved.
+#
+# Licensed under the Universal Permissive License v 1.0 as shown at 
+# http://oss.oracle.com/licenses/upl.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle SOA Suite
+#
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# See *.download file in this directory
+#
+# Pull base image
+# ---------------
+FROM oracle/fmw-infrastructure:12.2.1.3
```

Comment:
This image doesn't exist. The GitHub repo only has a `Dockerfile` for `fmw-infrastructure:12.2.1.2`.